aca_lts_eval.py

```diff
@@ -112,13 +112,18 @@ def get_options():
 
 
 def max_temp(time, stars):
+    # Sort these by mag to assign big boxes to first 3
+    stars.sort('MAG_ACA')
+    halfwidths = np.repeat(120, len(stars))
```

Comment:
Here, I'm really not sure if we should use the 120 arcsec halfwidth boxes or use any reduced box sizes that come out of the search algorithm (boxes reduced for bright stars, hot pixels, or edge of CCD).  I suppose we could have mini_sausage just adjust boxes for star spoilers (ignoring CCD things like hot pixels and CCD edge) so adjusted acq boxes would be roll-independent.  Basically off in the weeds anyway but we are getting more and more pushback when "close" isn't "close enough".

Reply:
120 for the other boxes seems fine for now.  proseco will be here soon.